app/src/addr.c

```diff
@@ -36,11 +39,22 @@ zxerr_t addr_getItem(int8_t displayIdx,
                      uint8_t pageIdx, uint8_t *pageCount) {
     zemu_log_stack("addr_getItem");
     switch (displayIdx) {
-        case 0:
-            snprintf(outKey, outKeyLen, "Pub Key");
-            pageString(outVal, outValLen, (char *) (G_io_apdu_buffer + VIEW_ADDRESS_OFFSET_SECP256K1), pageIdx, pageCount);
+        case 0: {
+            snprintf(outKey, outKeyLen, "Account");
+            array_to_hexstr(outVal, outValLen, G_io_apdu_buffer, sizeof(flow_account_t));
             return zxerr_ok;
+        }
         case 1: {
+            snprintf(outKey, outKeyLen, "Pub Key");
+            pageString(outVal, outValLen, (char *) (G_io_apdu_buffer + sizeof(flow_account_t) + VIEW_ADDRESS_OFFSET_SECP256K1), pageIdx, pageCount);
+            return zxerr_ok;
+        }
+        case 2: {
+            snprintf(outKey, outKeyLen, "Warning");
+            pageString(outVal, outValLen, "Ledger does not verify if acoount and pub key match!!!", pageIdx, pageCount);
```

Comment:
typo: account

Reply:
maybe this should go as first screen? :thinking:

Reply:
This is up to flow guys to decide.

Reply:
I suggest the following warning message:
```
"Ledger does not check if the on-chain account includes the public key!"
```

Reply:
i used "pub key" instead of "public key" so it fits within two ledger pages.